wordstats/language_data/t2l/code/README.md

```diff
@@ -0,0 +1,62 @@
+### cleaning_words.py
+
+If it is required to add words of a new language it is needed to follow the next steps:
+
+1. Download the aspell dictionary of that language (http://ftp.gnu.org/gnu/aspell/dict/) and follow the instructions. Basically you have to download the .tar.bz2 file (the 5 is the newest) and then:
+
+	-Unzip the folder
+
+	-Go with the terminal to that folder.
+
+	-Execute ./configure
+
+	-Execute make
+
+	-Execute make install
+
+	-Execute preunzip 'lang'.cwl (e.g: es.cwl)
+
+You will obtain a .wl file with the dictionnary.
+
+However there will be a script which will do this automatically, but if the result is not what you expect, then you can do it manually.
+
+2. It is needed to create a new .txt file and copy the dictionary. Python does not recognize .uw files. After that move the dictionary to the Dictionary folder.
+
```

Comment:
what is the Dictionary folder ?
i guess you mean 

    ../data/dictionaries/

I assume the current folder to be the folder in which the README file resides. 

which brings me to, wouldn't it make sense to move the README on the same level with `code` and `data`? 

then you talking about relative folders and files would be simpler. 

and also, the readme i imagine talks also about the data.